bcda/web/middleware.go

```diff
@@ -52,6 +52,7 @@ func HSTSHeader(next http.Handler) http.Handler {
 	return http.HandlerFunc(func(w http.ResponseWriter, r *http.Request) {
 		if servicemux.IsHTTPS(r) {
 			w.Header().Set("Strict-Transport-Security", "max-age=31536000; includeSubDomains; preload")
+			w.Header().Set("Cache-Control", "must-revalidate")
```

Comment:
This is good.  However, it slightly changes the purpose of the function, which is named after the other header set in the line above.  Shall we rename the function to `SecurityHeaders()` or something similar?

Reply:
Also, if you look at the scanner output attached to the original ticket, it's requesting a second header: `Pragma: no-cache`.  Let's add a line for that one, too.